packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx

```diff
@@ -91,4 +93,18 @@ export const PaymentLegalBlurb = () => (
   </div>
 );
 
+export type PaymentLegalBlurbProps = {
+  provider: PaymentProvider.ProviderType;
+};
+
+export const PaymentLegalBlurb = ({ provider }: PaymentLegalBlurbProps) => {
+  return (
+    <>
+      {PaymentProvider.isPaypal(provider) && <PaypalPaymentLegalBlurb />}
+      {PaymentProvider.isStripe(provider) && <StripePaymentLegalBlurb />}
+      {PaymentProvider.isNotChosen(provider) && <DefaultPaymentLegalBlurb />}
+    </>
```

Comment:
Tiny suggestion: if we group the `&&`s and put `||`s between them, then we can return early.  i.e.

```
  return (
    (PaymentProvider.isPaypal(provider) && <PaypalPaymentLegalBlurb />) ||
    (PaymentProvider.isStripe(provider) && <StripePaymentLegalBlurb />) ||
    (PaymentProvider.isNotChosen(provider) && <DefaultPaymentLegalBlurb />)
  );
```